tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -516,20 +516,22 @@ func TestRateLimit(t *testing.T) {
 	totalReqs := res.DurationHistogram.Count
 	succReqs := float64(res.RetCodes[http.StatusOK])
 	badReqs := res.RetCodes[http.StatusBadRequest]
+	actualDuration := res.ActualDuration.Seconds() // can be a bit more than requested
 
 	glog.Info("Successfully sent request(s) to /productpage; checking metrics...")
-	t.Logf("Fortio Summary: %d reqs (%f 200s (%f rps), %d 400s)", totalReqs, succReqs, succReqs/opts.Duration.Seconds(), badReqs)
+	t.Logf("Fortio Summary: %d reqs (%f rps, %f 200s (%f rps), %d 400s)",
+		totalReqs, res.ActualQPS, succReqs, succReqs/actualDuration, badReqs)
 
 	// consider only successful requests (as recorded at productpage service)
 	callsToRatings := succReqs
 
 	// the rate-limit is 1 rps
-	want200s := opts.Duration.Seconds()
+	want200s := 1. * opts.Duration.Seconds()
```

Comment:
instead of `int (opts.Duration.Seconds())` ? why this change?

Reply:
I wanted to make it a bit clearer it was 1 per second